addressfield_geosuggest.module

```diff
@@ -109,7 +109,26 @@ function addressfield_geosuggest_field_widget_form(&$form, &$form_state, $field,
     'element_id' => $css_id,
     'values_element_id' => $values_element_id,
     'default_values' => [],
-  ] + $instance['widget']['settings'];
```

Comment:
Can you add .gitattributes same as in recruiter_internal to get syntax highlighting on github?